src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const root = document.querySelector('body');
+
+const firstPromise = new Promise((resolve, reject) => {
+  root.addEventListener('click', () => (
+    resolve('First promise was resolved')
+  ));
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  root.addEventListener('click', () => (
+    resolve('Second promise was resolved')
+  ));
+
+  root.addEventListener('contextmenu', () => (
+    resolve('Second promise was resolved')
+  ));
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let left = false;
+  let right = false;
+
+  root.addEventListener('mouseup', (ev) => {
```

Comment:
you can also handle events, that will occur on left and right clicks, instead of handling separate buttons